tgui/packages/tgui/styles/interfaces/LobbyMenu.scss

```diff
@@ -0,0 +1,182 @@
+.theme-crtgreen {
+  .LobbyScreen {
+    &.filterEnabled {
+      box-shadow: 0 0 15px;
+    }
+
+    .typeEffect {
+      overflow: hidden;
+      white-space: nowrap;
+      width: 0;
+      animation: typing 1s steps(20, end) forwards;
+      animation-delay: 1.3s;
+    }
+
+    .buttonEffect {
+      overflow: hidden;
+      white-space: nowrap;
+      width: 0;
+      animation: typing 1s steps(20, end) forwards;
+    }
+
+    .loadEffect {
+      height: 0;
+      animation: loading 1s steps(20, end) forwards;
+      animation-delay: 1.3s;
+    }
+
+    .sectionLoad {
+      animation: section-loading 1s forwards;
```

Comment:
Might be a good idea to allow players to disable animations if they want a more snappy experience. Complex animations can generally be slow and I've noticed the UI lagging when I hover over buttons, etc